### PR TITLE
Update Panoptic nuScenes Eval readme

### PR DESCRIPTION
**Pull request by lubing-motional**
- Update Panoptic Eval readme on the evaluation server and primary ranking metrics.

### CODE REVIEW

File: python-sdk/nuscenes/eval/panoptic/README.md [hunk 1]
```diff
@@ -220,22 +234,18 @@ defined as: (∑{**1**(p,g) IoU(p,g)} - ∑(s)∈ IDS {s}) / (|TP|+ 0.5|FP|+ 0.5
...
-We also use LSTQ metric [Mehmet et al](https://arxiv.org/pdf/2102.12472.pdf). The LSTQ metric is computed as a geometric
-mean of the classification score and association score.
-
 ## Leaderboard
 nuScenes will maintain a single panoptic leaderboard with filters to split 4 specific tracks: Segmentation-lidar,
 Segmentation-open, Tracking-lidar and Tracking-open. Submissions of the first two panoptic segmentation tracks will be
-evaluated with segmentation metrics, while the two tracking submissions will be evaluated with both multi-object
-panopic tracking metrics as well as frame based panoptic segmentation metrics. For each submission the leaderboard will
+evaluated with segmentation metrics, while the two tracking submissions will be evaluated with both panopic tracking
+metrics as well as frame based panoptic segmentation metrics. For each submission the leaderboard will
```
Comment (whyekit-motional) [inline python-sdk/nuscenes/eval/panoptic/README.md:241]: `panopic tracking` --> `panoptic tracking`

File: python-sdk/nuscenes/eval/panoptic/README.md [hunk 2]
```diff
@@ -205,9 +207,21 @@ We also use PQ† [Porzi et al.](https://arxiv.org/pdf/1905.01220.pdf), which ma
...
+
+#### Panoptic Tracking (PAT)
+We define the Panoptic Tracking (PAT) metric, which is based on two separable components that are explicitly related to
+the task and allow straightforward interpretation. PAT is computed as the harmonic mean of the Panoptic Quality (PQ)
+and Tracking Quality (TQ): (2 x PQ x TQ) / (PQ + TQ), with range [0, 1]. To better represent the tracking quality,
+TQ is comprised of association score and track fragmentation components. Interested readers can refer to the
+[Panoptic nuScenes paper](https://arxiv.org/pdf/2109.03805.pdf) for more details of PAT metric.
+
+### LiDAR Segmentation and Tracking Quality (LSTQ)
+We also use LSTQ metric [Mehmet et al](https://arxiv.org/pdf/2102.12472.pdf). The LSTQ metric is computed as a geometric
```
Comment (whyekit-motional) [inline python-sdk/nuscenes/eval/panoptic/README.md:220]: We also use `the` LSTQ metric?

**Review by whyekit-motional — APPROVED** [review]
Looks good overall! Just a couple of very minor comments

File: python-sdk/nuscenes/eval/panoptic/README.md (hunk 2 above)
Comment (lubing-motional) [reply python-sdk/nuscenes/eval/panoptic/README.md]: modified

File: python-sdk/nuscenes/eval/panoptic/README.md (hunk 1 above)
Comment (lubing-motional) [reply python-sdk/nuscenes/eval/panoptic/README.md:241]: modified

File: python-sdk/nuscenes/eval/panoptic/README.md [hunk 3]
```diff
@@ -186,9 +185,12 @@ For more information on the classes and their frequencies, see
 
 
 ## Evaluation metrics
-Below we define the metrics for the Panoptic Segmentation and Multi-Object Panoptic Tracking tasks.
+Below we introduce the key metrics for panoptic segmentation and panoptic tracking tasks. We use Panoptic Quality (PQ)
+as the primary ranking metric for panoptic segmentation tasks, and Panoptic Tracking (PAT) metric as the primary
+ranking metric for panoptic tracking tasks.
```
Comment (holger-motional) [inline python-sdk/nuscenes/eval/panoptic/README.md:190]: To be discussed

File: python-sdk/nuscenes/eval/panoptic/README.md [hunk 4]
```diff
@@ -205,63 +207,71 @@ We also use PQ† [Porzi et al.](https://arxiv.org/pdf/1905.01220.pdf), which ma
...
+evaluated with segmentation metrics, while the two tracking submissions will be evaluated with both panoptic tracking
+metrics as well as frame based panoptic segmentation metrics. For each submission the leaderboard will
 list method aspects and evaluation metrics. Method aspects include input modalities (lidar, radar, vision), use of map
 data and use of external data. To enable a fair comparison between methods, the user will be able to filter the methods
 by method aspects.
 
-Both panoptic segmentation and multi-object panoptic tracking tasks will have an `Lidar track` and `Open Track`
-respectively. Methods will be compared within these tracks and the winners will be decided for each track separately.
-Furthermore, there will also be an award for novel ideas, as well as the best student submission.
+Both panoptic segmentation and panoptic tracking tasks will have an `Lidar track` and `Open Track` respectively.
```
Comment (holger-motional) [inline python-sdk/nuscenes/eval/panoptic/README.md:246]: an -> a

File: python-sdk/nuscenes/eval/panoptic/README.md [hunk 5]
```diff
@@ -205,63 +207,71 @@ We also use PQ† [Porzi et al.](https://arxiv.org/pdf/1905.01220.pdf), which ma
...
+Methods will be compared within these tracks and the winners will be decided for each track separately. Furthermore,
+there will also be an award for novel ideas, as well as the best student submission.
```
Comment (holger-motional) [inline python-sdk/nuscenes/eval/panoptic/README.md:248]: Let's remove `Furthermore, there will also be an award for novel ideas, as well as the best student submission.`

File: python-sdk/nuscenes/eval/panoptic/evaluate.py [hunk 6]
```diff
@@ -1,9 +1,9 @@
 """
 Code written by Motional and the Robot Learning Lab, University of Freiburg.
 
-Script to evaluate nuScenes-panoptic panoptic segmentation (PS) or multi-object panoptic tracking metrics (MOPT).
+Script to evaluate Panoptic nuScenes panoptic segmentation (PS) or panoptic tracking (PT) metrics.
 Argument "task" could be one of ["segmentation", "tracking"], check eval/panoptic/README.md for more details of the
-tasks. Note tracking results will be evaluated with both the MOPT and PS metrics.
+tasks. Note tracking results will be evaluated with both the PT and PS metrics.
```
Comment (holger-motional) [inline python-sdk/nuscenes/eval/panoptic/evaluate.py:6]: PT -> PAT?

**Review by holger-motional — APPROVED** [review]

File: python-sdk/nuscenes/eval/panoptic/evaluate.py (hunk 6 above)
Comment (lubing-motional) [reply python-sdk/nuscenes/eval/panoptic/evaluate.py:6]: PT stands for panoptic tracking, PS for panoptic segmentation, will use the full names.

**Pull request merged**